openstack/loadbalancer/v2/quotas/results.go

```diff
@@ -0,0 +1,46 @@
+package quotas
+
+import "github.com/gophercloud/gophercloud"
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// Extract is a function that accepts a result and extracts a Quota resource.
+func (r commonResult) Extract() (*Quota, error) {
+	var s struct {
+		Quota *Quota `json:"quota"`
+	}
+	err := r.ExtractInto(&s)
+	return s.Quota, err
+}
+
+// GetResult represents the result of a get operation. Call its Extract
+// method to interpret it as a Quota.
+type GetResult struct {
+	commonResult
+}
+
+// Quota contains load balancer quotas for a project.
+type Quota struct {
+	// Loadbalancer represents the number of load balancers. A "-1" value means no limit.
+	Loadbalancer int `json:"load_balancer"`
```

Comment:
According to https://github.com/openstack/octavia/blob/bd0206a6ea4efa342fe73e0d099238ca78ca1208/releasenotes/notes/Correct-naming-for-quota-resources-8e4309a839208cd1.yaml, `load_balancer` and `health_monitor` were deprecated 2 years ago.

This is also noted throughout the quotas type code (https://github.com/openstack/octavia/blob/master/octavia/api/v2/types/quotas.py#L25).

Can you confirm if this is just for the objects named throughout the Python code and not actually the JSON response? In other words, are you seeing both `load_balancer` and `loadbalancer` being returned?

If so, we'll want to change this to `loadbalancer` so we use the future-proof version.

Reply:
You were totally right. Our OpenStack does currently provide both, also another one having Queens release works fine using `loadbalancer` :-)

Reply:
Fixed in [55e3ace](https://github.com/gophercloud/gophercloud/pull/2010/commits/55e3acef5280c52616a243c962023b932d9f3f23)